packages/custom-props-layer/src/types.ts

```diff
@@ -1,9 +1,22 @@
-import { CustomPropsTreeOptions } from "@atrilabs/app-design-forest/lib/customPropsTree";
+import {
+  CustomPropsTreeOptions,
+  MapCustomProp,
+} from "@atrilabs/app-design-forest/lib/customPropsTree";
 import {
   ColorPickerDialogProps,
   OpenAssetManagerCallabck,
 } from "@atrilabs/shared-layer-lib";
 
+type attributeTypes =
```

Comment:
Derive this type like this:

```tsx
import { MapCustomProp } from "packages/app-design-forest/src/customPropsTree.ts";

type AttributeTypes = MapCustomProp["attributes"]
```

**NOTE: The type name must start with capital letter.**

Reply:
Done